### ISSUE TITLE
Bug - bulk - list generator - cali

### ISSUE THREAD

**Issue by jannellstearns** [MN-Script-Team/DHS-PRISM-Scripts#806]
when running this script today on a large caseload, it ran and then started looping on the same screen, not f8'ing and leaving blank lines in the spreadsheet.  I had used this script last month without issue.  I went through the cali list and it appears to be looping on the last screen continually and not stopping.  if I can provide more information, let me know.

**Comment by ChrisKast** [comment]
I have been playing with this script and noticed some errors with it as well.  It also seems to F8 or "transmit" when looking at the last payment therefore going down to the next screen of payments and not listing the most recent payment.  In addition it seems to add one final additional line of text on the excel spreadsheet that is a duplicate of the 2nd to last line. 

On a side note I'm trying to manipulate this script to include CP/NCP home and cell phone numbers.  VC was helping me with this last Thursday and might recall the errors we were receiving.

**Comment by theVKC** [comment]
I think there's something goofy in how it's reading the data, specifically regarding when the list-of-cases is complete. @ChrisKast: our issue that we saw, which looked like information from one case being copied over across many, echoes this specific issue... perhaps the original code itself (which we based our phone number modifications around) had a bug?

 @ChrisKast, does your "add the phone numbers" enhancement have a GitHub issue? If not, let's add it to this issue and we can fix both at the same time. I'd be happy to look at it with you, but would need access to the caseload you worked on, so we should probably do it over WebEx instead of through GitHub comments. :)

**Comment by heatherallen** [comment]
@theVKC I think the original bug should go on the "do-before-conversion" list if the script is not working correctly.

**Comment by ChrisKast** [comment]
True be told, I had the bug fixed but then tried to add some additional functionality to the script and that caused some additional bugs.  I’ll try to figure out what I did to fix the original bug.  I know my Supervisor would very much like for me to get the additional fuctionality part to work……I’m trying to get it to dump NCP and CP work/cell phone numbers onto the Excel spreadsheet.  Could we maybe work on this at the 2/27 or 3/8 Hackathon?  I’ll need some help…..most of the folks at the Hackathon’s are “getting their master’s” in script writing and I’m still stuck in “grade school”.

**Comment by theVKC** [comment]
I agree that we should get this done before conversion, and @ChrisKast, I would be happy to help at hackathon!

**Comment by theVKC** [comment]
@ChrisKast, @KallistaImdieke is going to reach out to you to see if we can parse the "bug fix" from the "enhancement", I'd like to get the bug fix out ASAP so we can have all bugs fixed in PRISM.

**Comment by KallistaImdieke** [comment]
@ChrisKast Do you have the code for this that I can look at to just get the bug fixed?  THANKS!
